codegen/src/index.ts

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env node
 import { program } from 'commander';
 import { join } from 'path';
+import ora from 'ora';
+import { access as lameCallbackBasedAccess } from 'fs';
+import { promisify } from 'util';
 
 import { generate } from './codegen';
+import { relative } from 'path/posix';
```

Comment:
I see that we are using `join` from `path` instead of `path/posix`, let's merge the import statements!